src/TodoList.js

```diff
@@ -1,5 +1,12 @@
 class TodoList {
-
+    
+    create(text) {
+        return {
+            id: 1,
+            text: text,
+            status: false
+        }
+    }
```

Comment:
Nice structure. The test should be failing now but you just need a small change to fix it. The status key of the object you're returning is storing a boolean and your test is looking for a string there.